tenants/README.md

```diff
@@ -31,6 +35,8 @@ The `resource-category` can be any logical grouping of resources that you wish t
 
 The `cluster` should correspond to the resources you wish to deploy to that specific cluster.  If you do not include a cluster in the overlay, those resources will not be synced to that cluster.
 
+Alternatively, you can utilize the `common` overlay name to allow the same configuration to be applied to all clusters instead of creating an individual cluster folder in the overlays folder.
```

Comment:
Can we add a quick note here that says 

"You can ONLY use one of or the other, i.e. you can't have like a rhoai-stable-2.10 and a common folder"

Just to make sure that is clear to folks